app_user/forms.py

```diff
@@ -54,17 +54,22 @@ def save(self, commit=True):
         return user
 
     def sendConfirmEmail(self, request, user):
-        confirmId = get_random_string(length=32)
-        link = 'http://' + HttpRequest.get_host(request) + reverse('app_user:confirm_email', kwargs={'uuid': confirmId})
-        emailMessage = 'Sehr geehrte/er Frau / Herr '+ request.POST.get('last_name') + ', <br><br>vielen dank für die Registrierung in book².<br> Zur Bestätigung Ihrer E-Mail Adresse betätigen Sie bitte folgenden Link: <a href="' + link + '">Bestätigen</a><br>Sollten Sie den Link nicht nutzen könnten dann kopieren Sie bitte folgende URL in Ihren Browser:<br> ' + link + '<br><br>Wir wünschen Ihnen viel Spaß beim Shoppen<br>Ihr book² team'
-
-
+        try:
+            confirmEmail = ConfirmEmail.objects.get(user=user)
+        except ConfirmEmail.DoesNotExist:
+            confirmId = get_random_string(length=32)
+            try:
+                user = User.objects.get(email=self.request.POST.get('email'))
```

Comment:
Kann übrigens auch als `user = User.objects.filter(email=self.request.POST.get('email')).first()` geschrieben werden. Dann ersparst du dir das Exception-Handling in diesem Fall